### ISSUE TITLE
loops on/off with midi controller

### ISSUE THREAD

**Issue by y1ds** [ahlstromcj/sequencer64#35]
I`m having problems turning on and off loops with a controller.
First i tried to use the config file that i used for seq24, and was working.
I realised that the "turn on" and "turn off" in the bindings in the rc file are the other way around, also turning on did not work. I can only turn a loop off.
When i run sequencer64 in legacy mode, the on/off is not inverted anymore, but still, only turning a loop off works.

**Comment by ahlstromcj** [comment]
I am looking into this.  It's a little bit complex, and may take me awhile to figure out what's wrong here.
Thanks for the report!

**Comment by ahlstromcj** [comment]
I found a couple bugs related to this issue, but I think I still have more to investigate.  You can checkout the latest in the fixups branch if you want to try the new code.

**Comment by y1ds** [comment]
I tried the fixup branch but im sorry to say, it got worse...  now also loops are not turning off anymore.
also, when you quit seq64, all the midi bindings from the sequencer64.rc file are removed... Haven`t really had any time to look into it but I will try to help finding the problem.

**Comment by ahlstromcj** [comment]
Okay, will keep looking.  Once the hurricane passes :-*

**Comment by ahlstromcj** [comment]
Actually working in a hotel right now.  I found a real bonehead line of code that was checking the sequence number against the wrong range when loading up and accessing the MIDI control values arrays.  So essentially nothing was getting loaded (and thus wiped out when saved, and not working because empty.  The only reason it partly worked before was because of the bug I fixed earlier.  Anyway, I have only had a chance to check the loading and saving of the MIDI controls, so if you have a chance to let me know how much works (or is still borken) that would be great!
